src/api/layer/profit_and_loss.ts

```diff
@@ -6,20 +6,41 @@ import type {
   ProfitAndLossSummaries,
 } from '../../types/profit_and_loss'
 import { get, post } from './authenticated_http'
+import { toLocalDateString } from '../../utils/time/timeUtils'
 
-export const getProfitAndLoss = get<{
-  data?: ProfitAndLoss
-  error?: unknown
-}>(
-  ({ businessId, startDate, endDate, tagKey, tagValues, reportingBasis }) =>
-    `/v1/businesses/${businessId}/reports/profit-and-loss?start_date=${
-      startDate ? encodeURIComponent(startDate) : ''
-    }&end_date=${endDate ? encodeURIComponent(endDate) : ''}${
+type BaseProfitAndLossParams = {
+  businessId: string
+  startDate?: Date
+  endDate?: Date
+  month?: string
+  year?: string
+  tagKey?: string
+  tagValues?: string
+  reportingBasis?: string
+}
+
+type GetProfitAndLossParams = BaseProfitAndLossParams
+type GetProfitAndLossCsvParams = BaseProfitAndLossParams & {
+  moneyFormat?: string
+}
+
+export const getProfitAndLoss = (apiUrl: string, accessToken: string | undefined, params: GetProfitAndLossParams) => {
+  const { businessId, startDate, endDate, month, year, tagKey, tagValues, reportingBasis } = params
+  return get<{
+    data?: ProfitAndLoss
+    error?: unknown
+  }>(({ businessId }) =>
+    `/v1/businesses/${businessId}/reports/profit-and-loss?${
+      startDate ? `start_date=${encodeURIComponent(toLocalDateString(startDate))}` : ''
+    }${endDate ? `&end_date=${encodeURIComponent(toLocalDateString(endDate))}` : ''}${
```

Comment:
The API supports year/month here in addition to start_date and end_date.
We don't actually use the year/month anywhere in the components today, but I wanted to refactor this and the CSV download to 
1. Accurately reflect the API endpoint parameters
2. Share the same types

Reply:
But this change isn't actually necessary / related to the timestamp clamping